open_tms_backend/open_tms_add_user_to_organisation/index.js

```diff
@@ -0,0 +1,141 @@
+import { shared } from '@appblocks/node-sdk'
+import { v4 as uuidv4 } from 'uuid'
+
```

Comment:
Please use examples/list_tickets as reference and add swagger documentation here to the handler so that this can be tested via swagger.Also refer to contribution.md.